src/app/series/directives/series-podcast.component.ts

```diff
@@ -66,27 +55,35 @@ export class SeriesPodcastComponent implements OnDestroy, DoCheck {
 
   ngOnDestroy(): any {
     this.tabSub.unsubscribe();
-    if (this.templateSub) {
-      this.templateSub.unsubscribe();
-    }
   }
 
-  loadPodcast() {
-    this.podcast = null;
-    if (this.distribution) {
-      this.distribution.loadExternal().subscribe(() => this.setPodcast);
+  loadDistributions() {
+    let initial = this.initialDistributions;
```

Comment:
Was having some issues with change detection here.  So ended up recording what the initial `series.distributions` array looked like, and manually triggering changes when it is different.